examples/applications/plot_cyclical_feature_engineering.py

```diff
@@ -0,0 +1,741 @@
+
+"""
+==========================
+Cyclic feature engineering
```

Comment:
I don't know if we should have something more related to "date-time encoding". I might think it might be easier to find than "cyclic" even if the title is correct.

Reply:
I changed the title. The title and the filename no longer match though. I think it's ok but not 100% sure.